readme.md

```diff
@@ -1,34 +1,38 @@
 Wagtail demo project
 =======================
 
-This is a demonstration project for [Wagtail CMS](http://wagtail.io).
+This is a demonstration project for the amazing [Wagtail CMS](https://github.com/wagtail/wagtail).
 
-*We do __not__ recommend using this project to start your own site*. This project is only to provide some examples of
-implementing common features, it is not an exemplar of Django or Wagtail best practice.
+The demo site is designed to provide examples of common features and recipes to introduce you to Wagtail development. Beyond the code, it will also let you explore the admin and editorial interface of the CMS.
 
-This project can be installed in one of three ways:
+Note we do _not_ recommend using this project to start your own site - the demo is intended to be a springboard for you to get started. Feel free to copy code from the demo into your own project.
 
-- Vagrant
-- Docker
-- Traditional/manual Django setup
+**Document contents**
 
-If you're new to Python/Django, we suggest you run this project on a Virtual Machine using Vagrant or
-via Docker, both of which help resolve common software dependency issues. Developers more familiar with
```

Comment:
Vagrant is definitely an easier way to bring up a build than virtualenv. To use virtualenv one requires a reasonably good knowledge of how Python is running on your MacOSX and can cause software conflicts/problems if done incorrectly (because it's using global dependencies). Specific to this project if a user doesn't have a non-system version of Python (e.g. via Pyenv or Brew), PIP, Postgres etc all installed (or if something's gone wrong with their `$PATH` variable) they'll find it frustrating bringing up a build.

Since Vagrant (and to an extent Docker) specify the whole machine it's much more self-contained and entirely agnostic about the host computer, which should make it easier for a new user to implement it. It should be a case of `vagrant up` and waiting a while. Speaking for myself, even though each Virtualbox VM can be several GB, conceptually I still much prefer Vagrant. It I destroy a vagrant box I know it's gone. In any case given that we're offering four ways to bring up a build I'm not sure we're prejudicing people if they want to use virtualenv to bring up the demo, though happy to tweak the language if you think we should?